captum/optim/_utils/models.py

```diff
@@ -1,322 +1,326 @@
+        return F.relu(input, inplace=self.inplace)
+
+
+def replace_layers(
+    model: nn.Module,
+    layer1: Type[nn.Module],
+    layer2: Type[nn.Module],
+    transfer_vars: bool = False,
+    **kwargs
+) -> None:
+    """
+    Replace all target layers with new layers inside the specified model,
+    possibly with the same initialization variables.
+
+    Args:
+        model: (nn.Module): A PyTorch model instance.
+        layer1: (nn.Module): A layer instance that you want to transfer
+            initialization variables from.
+        layer2: (nn.Module): The layer class to create with the variables
+            from of layer1.
+        transfer_vars (bool, optional): Wether or not to try and copy
+            initialization variables from layer1 instances to the replacement
+            layer2 instances.
+        kwargs: (Any, optional): Any additional variables to use when creating
+            the new layer.
+    """
+
+    for name, child in model._modules.items():
+        if isinstance(child, layer1):
+            if transfer_vars:
+                new_layer = _transfer_layer_vars(child, layer2, **kwargs)
+            else:
+                new_layer = layer2(**kwargs)
+            setattr(model, name, new_layer)
+        elif child is not None:
+            replace_layers(child, layer1, layer2, transfer_vars, **kwargs)
+
+
+def _transfer_layer_vars(
+    layer1: nn.Module, layer2: Type[nn.Module], **kwargs
+) -> nn.Module:
+    """
+    Given a layer instance, create a new layer instance of another class
+    with the same initialization variables as the original layer.
+    Args:
+        layer1: (nn.Module): A layer instance that you want to transfer
+            initialization variables from.
+        layer2: (nn.Module): The layer class to create with the variables
+            from of layer1.
+        kwargs: (Any, optional): Any additional variables to use when creating
+            the new layer.
+    Returns:
+        layer2 instance (nn.Module): An instance of layer2 with the initialization
+            variables that it shares with layer1, and any specified additional
+            initialization variables.
+    """
+
+    l2_vars = list(signature(layer2.__init__).parameters.values())
+    l2_vars = [
+        str(l2_vars[i]).split()[0]
+        for i in range(len(l2_vars))
+        if str(l2_vars[i]) != "self"
+    ]
+    l2_vars = [p.split(":")[0] if ":" in p and "=" not in p else p for p in l2_vars]
+    l2_vars = [p.split("=")[0] if "=" in p and ":" not in p else p for p in l2_vars]
+    layer2_vars: Dict = {k: [] for k in dict.fromkeys(l2_vars).keys()}
+
+    layer1_vars = {k: v for k, v in vars(layer1).items() if not k.startswith("_")}
+    shared_vars = {k: v for k, v in layer1_vars.items() if k in layer2_vars}
+    new_vars = dict(item for d in (shared_vars, kwargs) for item in d.items())
+    return layer2(**new_vars)
+
+
+class LocalResponseNormLayer(nn.Module):
+    """
```

Comment:
Originally I was getting errors about the weights not matching the class which is why I made wrappers for the functional versions of some classes. I just tested using `torch.nn.LocalResponseNorm` & `torch.nn.ReLU` instead of their wrapper classes (`LocalResponseNormLayer`, `ReLULayer`), and it seemed to work. If there's an error about it not matching then we can just change the model file itself.

Reply:
I think that we can just replace the `LocalResponseNormLayer` and `ReLULayer` layers with `torch.nn.LocalResponseNorm` & `torch.nn.ReLU`. We can also remove those two wrapper layers from model utils along with their tests.

Reply:
Fixed this. Using torch.nn layers instead.